packages/superset-ui-legacy-preset-chart-big-number/src/BigNumber/transformProps.js

```diff
@@ -18,10 +18,45 @@
  */
 import * as color from 'd3-color';
 import { getNumberFormatter, NumberFormats } from '@superset-ui/number-format';
+import { getTimeFormatter, TimeFormats, smartDateVerboseFormatter } from '@superset-ui/time-format';
 import { renderTooltipFactory } from './BigNumber';
 
 const TIME_COLUMN = '__timestamp';
 
+function getTimeFormatterForGranularity(granularity) {
```

Comment:
This seems useful to extract into its own file, in case we move this to `@superset-ui/time-format` later